x-pack/plugins/observability_solution/synthetics/server/routes/monitor_cruds/add_monitor.ts

```diff
@@ -4,101 +4,112 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { v4 as uuidV4 } from 'uuid';
 import { schema } from '@kbn/config-schema';
-import {
-  SavedObject,
-  SavedObjectsClientContract,
-  KibanaRequest,
-  SavedObjectsErrorHelpers,
-} from '@kbn/core/server';
-import { isValidNamespace } from '@kbn/fleet-plugin/common';
-import { DEFAULT_SPACE_ID } from '@kbn/spaces-plugin/common';
-import { DefaultAlertService } from '../default_alerts/default_alert_service';
-import { triggerTestNow } from '../synthetics_service/test_now_monitor';
-import { SyntheticsServerSetup } from '../../types';
-import { RouteContext, SyntheticsRestApiRouteFactory } from '../types';
-import { syntheticsMonitorType } from '../../../common/types/saved_objects';
-import {
-  ConfigKey,
-  MonitorFields,
-  SyntheticsMonitor,
-  EncryptedSyntheticsMonitorAttributes,
-} from '../../../common/runtime_types';
-import { formatKibanaNamespace } from '../../synthetics_service/formatters/private_formatters';
-import { getPrivateLocations } from '../../synthetics_service/get_private_locations';
-import { PrivateLocationAttributes } from '../../runtime_types/private_locations';
+import { SavedObjectsErrorHelpers } from '@kbn/core/server';
+import { i18n } from '@kbn/i18n';
+import { InvalidLocationError } from '../../synthetics_service/project_monitor/normalizers/common_fields';
+import { AddEditMonitorAPI, CreateMonitorPayLoad } from './add_monitor/add_monitor_api';
+import { SyntheticsRestApiRouteFactory } from '../types';
 import { SYNTHETICS_API_URLS } from '../../../common/constants';
-import {
-  DEFAULT_FIELDS,
-  DEFAULT_NAMESPACE_STRING,
-} from '../../../common/constants/monitor_defaults';
-import { validateMonitor } from './monitor_validation';
-import { sendTelemetryEvents, formatTelemetryEvent } from '../telemetry/monitor_upgrade_sender';
-import { formatSecrets } from '../../synthetics_service/utils/secrets';
-import { deleteMonitor } from './delete_monitor';
+import { normalizeAPIConfig, validateMonitor } from './monitor_validation';
 import { mapSavedObjectToMonitor } from './helper';
 
 export const addSyntheticsMonitorRoute: SyntheticsRestApiRouteFactory = () => ({
   method: 'POST',
   path: SYNTHETICS_API_URLS.SYNTHETICS_MONITORS,
-  validate: {
-    body: schema.any(),
-    query: schema.object({
-      id: schema.maybe(schema.string()),
-      preserve_namespace: schema.maybe(schema.boolean()),
-      gettingStarted: schema.maybe(schema.boolean()),
-    }),
+  validate: {},
```

Comment:
Question: Should this route have access tags? or is there another means of authorization in place for these routes?

Reply:
yes, we have by default all routes mapped to uptime write tag